server/cmd/mmctl/commands/user_e2e_test.go

```diff
@@ -192,24 +209,130 @@ func (s *MmctlE2ETestSuite) TestListUserCmd() {
 		printer.Clean()
 
 		var page int
-		perpage := 10
+		perpage := 12
+		all := true
+		team := ""
+		var inactive bool
+		cmd := &cobra.Command{}
+		cmd.Flags().IntVar(&page, "page", page, "page")
+		cmd.Flags().IntVar(&perpage, "per-page", perpage, "perpage")
+		cmd.Flags().BoolVar(&all, "all", all, "all")
+		cmd.Flags().StringVar(&team, "team", team, "team")
+		cmd.Flags().BoolVar(&inactive, "inactive", inactive, "inactive")
```

Comment:
Is there a way to reuse `ListUsersCmd` instead of re-defining one?

Reply:
Yes. updated tests

Reply:
Actually, I wonder if it's possible to use `ListUsersCmd` directly given that it's a global variable.

Reply:
Owh ya. the latest changes does do that. ResetListUsersCmd() actually resets the flags for the command to be default. 
Then, custom flags are changed for the tests